### PR TITLE
Added cChunkDef::IsValidHeight()

### PR DESCRIPTION
**Pull request by SphinxC0re**

### CODE REVIEW

File: src/ChunkDef.h [hunk 1]
```diff
@@ -115,6 +115,11 @@ class cChunkDef
 		return Vector3i(a_RelBlockPosition.x + a_ChunkX * Width, a_RelBlockPosition.y, a_RelBlockPosition.z + a_ChunkZ * Width);
 	}
 
+	inline static bool IsValidHeight(int a_height)
+	{
+		return ((a_height >= 1) && (a_height <= Height));
```
Comment (madmaxoft) [inline src/ChunkDef.h]: Capitalization is wrong. `a_Height`

**Comment by madmaxoft** [comment]
See how easy coding is once you get the rage going? :D

**Comment by SphinxC0re** [comment]
I'm sick of inconsistency in height checks! Sick of it!

**Comment by madmaxoft** [comment]
Yessssss. Let the hate flow :D

**Comment by NiLSPACE** [comment]
Good Good

**Comment by Gargaj** [comment]
Shouldn't that be "< Height", since Height is 256?

**Comment by madmaxoft** [comment]
Yup. And that's why we're refactoring it out, because every third instance of this check is wrong.

**Comment by tigerw** [comment]
You've all gone mad.

**Comment by worktycho** [comment]
Isn't 0 a valid height?

**Comment by SphinxC0re** [comment]
Okay what do I have to check for? @madmaxoft

**Comment by SphinxC0re** [comment]
Is this mergeable? @cuberite/core-team

**Comment by NiLSPACE** [comment]
Perhaps we should export cChunkDef to Lua as well. For plugins that implement their own generator.

**Pull request merged**